test/tests/auxkernels/array_aux_kernels/function_array_aux.i

```diff
@@ -58,11 +62,6 @@
     variable = y
     functions = 'func1 func2'
   []
-  [z]
-    type = FunctionArrayAux
-    variable = z
-    functions = 'func1 func2 func3'
-  []
```

Comment:
duplicate the test and make sure they compare the same
use active = '' / inactive='' to have both syntaxes in the file (and avoid duplicating the file)